WalletWasabi.Backend/Controllers/BatchController.cs

```diff
@@ -53,7 +53,10 @@ public async Task<IActionResult> GetSynchronizeAsync([FromQuery, Required]string
 				}
 			}
 
-			var knownHash = new uint256(bestKnownBlockHash);
+			if (!uint256.TryParse(bestKnownBlockHash, out var knownHash))
+			{
+				return BadRequest("Invalid bestKnownBlockHash.");
```

Comment:
```suggestion
				return BadRequest($"Invalid {nameof(bestKnownBlockHash)}.");
```

Reply:
ACK for the suggestion (other than that it's good.)